src/__experimental__/components/search/search.spec.js

```diff
@@ -0,0 +1,214 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { act } from 'react-dom/test-utils';
+import { Input } from '../input';
+import Button from '../../../components/button';
+import Search from './search.component';
+import { StyledSearchButton } from './search.style';
+import { assertStyleMatch } from '../../../__spec_helper__/test-utils';
+import StyledTextInput from '../input/input-presentation.style';
+import Icon from '../../../components/icon';
+import { rootTagTest } from '../../../utils/helpers/tags/tags-specs';
+
+describe('Search', () => {
+  let wrapper, onBlur, onChange, onClick, onKeyDown;
+
+  const renderWrapper = (props, render = shallow) => (
+    render(
+      <Search
+        { ...props }
+      />
+    )
+  );
+  describe('styles', () => {
+    it('matches the expected styles', () => {
+      assertStyleMatch({
+        borderBottom: '1px solid #CCD6DA',
+        display: 'inline-flex',
+        fontSize: '14px',
+        fontWeight: '700'
+      }, renderWrapper({ value: '' }, mount));
+    });
+
```

Comment:
This can be in its own describe caluse.

`describe('matches the expected styles', () => {`

and then it blocks 

`it('when the input is focused', () => {`

Reply:
I take onboard your suggestion however, I felt that having the styles all in one `describe` block makes it easier to read and see the tests in the terminal.